Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -753,7 +798,8 @@ def parse_table_row_to_dict(alert: list[dict[str, Any]]) -> dict[str, Any]:
             continue
         obj[key] = base64.b64decode(value).decode("utf-8")
         # Remove " from the beginning and end of the string
-        obj[key] = obj[key].replace('"', "")
+        if key != "labels":
```

Comment:
is the lable key part from the get_user_context_by_email_address output?

Reply:
Yes

Reply:
is there any response with the label key? I saw that there are a lot of calls to this function and i would be happy to verify this with you

Reply:
Reco_test:377